examples/TestDriver/src/pages/basics.js

```diff
@@ -14,26 +14,36 @@ import {
 import { Switch as NbSwitch } from 'native-base';
 import { connect } from 'react-redux';
 
-import Heap from '@heap/react-native-heap';
+import Heap, { HeapIgnore } from '@heap/react-native-heap';
 import { incrementAction, decrementAction } from '../reduxElements';
 import { makeSentinelButton } from '../sentinelUtilities';
 
+const HeapIgnoredNbSwitch = Heap.withHeapIgnore(NbSwitch, {
+  ignoreInteraction: false,
+  ignoreAllProps: true,
+  ignoreTargetText: false,
+  ignoreInnerHierarchy: false,
+});
+
 class BasicsPage extends Component {
   render() {
     return (
       <View style={styles.container}>
+        <HeapIgnoredNbSwitch testID="ignoredNbSwitch" />
         <NbSwitch testID="nbSwitch" />
         <Switch testID="switch" />
         <Button
           testID="track1"
           title="Call Track1"
           onPress={() => Heap.track('pressInTestEvent1', {})}
         />
-        <Button
-          testID="track2"
-          title="Call Track2"
-          onPress={() => Heap.track('pressInTestEvent2', {})}
-        />
+        <HeapIgnore ignoreInteraction={false} ignoreTargetText={false}>
```

Comment:
Can you ignore the interaction (ignoreInteraction=true) but NOT ignore the targetText (targetText=false)... are there a limited number of configuration states that make sense?

Reply:
This is a good question (which would've been great on the RFC 😉).  Certain config props do imply other config props (regardless of the value of the other prop).  For instance, you can't capture hierarchy props if you're not capturing the inner hierarchy (since the props you'd be capturing would go in the inner hierarchy).  Similarly, if you're not capturing the interaction, you can't capture _anything else_, since there's no interaction to attach these other things (target text, hierarchy, props etc).